android/build.gradle

```diff
@@ -20,17 +35,22 @@ android {
 }
 
 repositories {
+    mavenLocal()
     maven {
         // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
-        // Matches the RN Hello World template
-        // https://github.com/facebook/react-native/blob/1e8f3b11027fe0a7514b4fc97d0798d3c64bc895/local-cli/templates/HelloWorld/android/build.gradle#L21
-        url "$projectDir/../node_modules/react-native/android"
+        url "$rootDir/../node_modules/react-native/android"
```

Comment:
And this?

Reply:
It's taken directly from the main `react-native` repository. I did not make any additional changes to the repos listed here. It's also the way plenty of other react native library repos are set up. If you have a different opinion, please let me know and I'll change it.